contracts/Sweep/TransferApprover/VestingApprover.sol

```diff
@@ -247,14 +199,6 @@ contract VestingApprover is ITransferApprover, Ownable {
         return vestingSchedules[beneficiary];
     }
 
-    /**
-     * @dev Checks if account is whitelisted
-     * @param account The address to check
-     */
-    function isWhitelisted(address account) external view returns (bool) {
-        return whitelists[account];
-    }
-
     /**
      * @dev Returns the current time.
      * @return the current timestamp in seconds.
```

Comment:
getCurrentTime is an unnecessary function. block.timestamp is used only once.

Reply:
I removed it